build/mount-proxy/Dockerfile

```diff
@@ -60,3 +66,28 @@ RUN ln -s /host/etc/mime.types /etc/mime.types
 COPY --link --from=builder /out/csi-mount-proxy* /usr/local/bin/
 ENTRYPOINT ["csi-mount-proxy-server", "--driver=ossfs"]
 
+#################### OSSFS 1.88 MOUNT-PROXY IMAGE ####################
+
+#################### ALINAS MOUNT-PROXY IMAGE ####################
+
+FROM registry-cn-hangzhou.ack.aliyuncs.com/dev/alinux:3-update as alinas
+
+ARG TARGETPLATFORM
+ARG ALINAS_UTILS_VERSION=1.8-0.20250228140224.8879f0.al7
+ARG EFC_VERSION=1.8-20250226151552.7151bf.release
+RUN set -ex; \
+    case "${TARGETPLATFORM}" in \
+        linux/amd64) ARCH="x86_64" ;; \
+        # linux/arm64) ARCH="aarch_64" ;; \
+        *) echo "unsupported platform"; exit 1 ;; \
+    esac; \
+    yum install -y util-linux procps tini; \
+    yum install -y https://aliyun-alinas-eac.oss-cn-beijing.aliyuncs.com/aliyun-alinas-utils-${ALINAS_UTILS_VERSION}.noarch.rpm; \
+    yum install -y https://aliyun-alinas-eac.oss-cn-beijing.aliyuncs.com/alinas-efc-${EFC_VERSION}.${ARCH}.rpm; \
+    yum clean all
+RUN cp -r /etc/aliyun /etc/aliyun-defaults
```

Comment:
What is the purpose of copying the binary?

Reply:
Add some explaination

Reply:
Added to comments